rholang/src/main/scala/coop/rchain/rholang/interpreter/SystemProcesses.scala

```diff
@@ -124,12 +124,12 @@ object SystemProcesses {
           } yield ()
       }
 
-      def getErrorMessagePar(str: String) =
+      private def getErrorMessagePar(str: String): Par =
         RevAddress
           .parse(str)
           .swap
           .toOption
-          .fold(Par.defaultInstance)(GString(_))
+          .fold(Par.defaultInstance)(GString)
```

Comment:
Please fixup the relevant commit with the `fix compilation warnings in SystemProcesses.scala` one